backend/src/schema/resolvers/posts.js

```diff
@@ -181,4 +182,46 @@ export default {
       return emoted
     },
   },
+  Post: {
+    ...Resolver('Post', {
+      hasMany: {
+        tags: '-[:TAGGED]->(related:Tag)',
+        categories: '-[:CATEGORIZED]->(related:Category)',
+        comments: '<-[:COMMENTS]-(related:Comment)',
+        shoutedBy: '<-[:SHOUTED]-(related:User)',
+        emotions: '<-[related:EMOTED]',
+      },
+      hasOne: {
+        author: '<-[:WROTE]-(related:User)',
+        disabledBy: '<-[:DISABLED]-(related:User)',
+      },
+      count: {
+        shoutedCount:
+          '<-[:SHOUTED]-(related:User) WHERE NOT related.deleted = true AND NOT related.disabled = true',
+        emotionsCount: '<-[related:EMOTED]-(:User)',
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:24:27Z" title="Friday, August 30th 2019, 2:24:27 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -179,4 +180,46 @@ export default {
       return emoted
     },
   },
+  Post: {
+    ...Resolver('Post', {
+      hasMany: {
+        tags: '-[:TAGGED]->(related:Tag)',
+        categories: '-[:CATEGORIZED]->(related:Category)',
+        comments: '<-[:COMMENTS]-(related:Comment)',
+        shoutedBy: '<-[:SHOUTED]-(related:User)',
+        emotions: '<-[related:EMOTED]',
+      },
+      hasOne: {
+        author: '<-[:WROTE]-(related:User)',
+        disabledBy: '<-[:DISABLED]-(related:User)',
+      },
+      count: {
+        shoutedCount:
+          '<-[:SHOUTED]-(related:User) WHERE NOT related.deleted = true AND NOT related.disabled = true',
+        emotionsCount: '<-[related:EMOTED]-(:User)',
```


I mentioned this in our pairing session, but I'm not sure if you heard/understood... I believe we are not using this anywhere in the code base